module.info

```diff
@@ -4,4 +4,4 @@ Description: Generic Trouble Ticket System connection module
  This module allows to easily link issue/ticket numbers in your Icinga
  acknowledgement or downtime comments. Configure a pattern matching
  your ticket numbers and a URL with placeholders. Please note that
- Icinga Web 2 version 2.3.0 is required at least to run this module.
+ Icinga Web 2 version 2.11.0 is required at least to run this module.
```

Comment:
Hi,
You do not have to change the version requirement for Icinga Web 2. This hook is also available in earlier versions and even in v2.3.0.